src/pages/FormPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styled from 'styled-components';
+import Heading from '../components/Heading';
+
+export default function FormPage() {
+  return (
+    <>
+      <Heading text={'What are you grateful for today?'} />
+    </>
```

Comment:
Same as above, if you are only returning one element, you do not need the fragments.

```suggestion
      <Heading text={'What are you grateful for today?'} />
```